cypress/e2e/supervision/deputies/unlink-deputy.cy.js

```diff
@@ -30,14 +30,16 @@ describe(
       cy.get(".TABS_DEPUTIES").click();
       cy.get("#deputies-table tr.summary-row").should("have.length", 2)
 
-      cy.get("@deputy").then(({salutation, firstname, surname}) => {
+      cy.get("@deputy").then(({ salutation, firstname, surname }) => {
+        cy.intercept({ method: "DELETE" }).as("unlinkDeputyCall");
```

Comment:
Would you mind explaining why you've changed this to an intercept?

Reply:
Before, it was clicking the Unlink button and immediately reloading the page, expecting the deputy to be unlinked. Sometimes the deputy was still there, either because the reload interrupted the unlink, or because the unlink request took a little longer than the reload.

By intercepting the unlink request, we can wait for its response (ensuring that the deputy has been unlinked), and only then reload the page and do the assertions

Reply:
That makes sense, thanks for explaining